static/js/shared/stats_var_labels.ts

```diff
@@ -290,6 +290,17 @@ const STATS_VAR_LABEL: { [key: string]: string } = {
   "WorldBank/SH_SVR_WAST_MA_ZS": "Male",
 
   "WorldBank/SI_POV_GINI": "Gini Index",
+
+  // Housing
+  Count_HousingUnit_HomeValueUpto49999USDollar: "0 - 50k",
```

Comment:
Should we add the unit to these labels? 0 - $50k

Reply:
Done